packages/web/package.json

```diff
@@ -6,7 +6,8 @@
   ],
   "main": "./dist/index.js",
   "exports": {
-    ".": "./dist/index.js",
+		".": "./dist/index.js",
```

Comment:
I'll fix the spacing, but @Tobbe  I had to add this here to get it working in typescript (internal to redwood). Might be useful in the future.